core-bundle/contao/dca/tl_page.php

```diff
@@ -683,7 +684,31 @@ static function ($value) {
 			),
 			'sql'                     => "int(10) unsigned NOT NULL default 0",
 			'relation'                => array('type'=>'hasOne', 'load'=>'lazy')
-		)
+		),
+		'enableCsp' => array
+		(
+			'inputType'               => 'checkbox',
+			'eval'                    => array('submitOnChange'=>true),
+			'sql'                     => array('type'=>'boolean', 'default'=>false),
+		),
+		'csp' => array
+		(
+			'inputType'               => 'text',
```

Comment:
Maybe use `textarea` here? I have the following CSP in one of my projects, which is quite cumbersome to manage in an one-line text field:

```
Content-Security-Policy: default-src 'self'; block-all-mixed-content; connect-src 'self' https://embed.domain.com https://upload.domain.com https://va.domain.com wss://*.domain.com; font-src 'self' data: https://embed.domain.com; frame-ancestors 'self'; img-src 'self' data: https://embed.domain.com https://cdn.jsdelivr.net; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://embed.domain.com https://cdn.jsdelivr.net; style-src 'self' 'unsafe-inline' https://embed.domain.com
```